api/imapsync/update

```diff
@@ -98,6 +98,19 @@ if ($cmd eq 'update') {
 } elsif ($cmd eq 'stop') {
     # stop the service
     system (qw(/usr/bin/systemctl  stop),  "imapsync\@$input->{'name'}");
+} elsif ($cmd eq 'start') {
+    # start the service
+    system (qw(/usr/bin/systemctl  start),  "imapsync\@$input->{'name'}");
+} elsif ($cmd eq 'synchronizeAll') {
+
+    my @Env = glob('/var/lib/nethserver/imapsync/*.env');
+
+    foreach my $name (@Env) {
+        # start the service
+        $name =~ s|/var/lib/nethserver/imapsync/||;
+        $name =~ s/\.env$//;
+        system (qw(/usr/bin/systemctl  start),  "imapsync\@$name");
+    }
```

Comment:
systemctl could start more units at once, if we are not afraid of concurrent fetches...

```suggestion
    system (qw(/usr/bin/systemctl  start),  map { m|^/var/lib/nethserver/imapsync/(.+)\.env$| ; "imapsync\@$1" } glob('/var/lib/nethserver/imapsync/*.env') );
```

Reply:
can we take into account that probably we could have a lot of configured account :D